tools/check_version.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
```

Comment:
Do we need two copies of this script?

Reply:
No, I checked in both copies by mistake. Only the one in tools should be needed (feel free to suggest an alternate location).